### PR TITLE
[FIX] tests case

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
Depends on: https://github.com/qrtl/sbtechnology/pull/54

### CODE REVIEW

**Comment by AungKoKoLin1997** [comment]
Finally I found the way how to use private repo in CI. The tests are successful now.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -59,7 +60,7 @@ jobs:
           repository: qrtl/sbtechnology
           token: ${{ secrets.CI_PRIVATE_REPO_ACCESS_TOKEN }}
           path: sbtechnology
-          ref: refs/heads/master
+          ref: refs/heads/12.0-upd-license
```
Comment (kanda999) [inline .github/workflows/test.yml]: Is this branch correnct?  I have merged https://github.com/qrtl/sbtechnology/pull/54

Comment (AungKoKoLin1997) [reply .github/workflows/test.yml]: I have to change if you have merged this branch.

**Comment by AungKoKoLin1997** [comment]
Closed this PR. The changes of this update is added in the latest dotfiles PR https://github.com/qrtl/sbt-custom/pull/9.

**Pull request closed without merging**